dom.bs

```diff
@@ -1270,7 +1270,7 @@ for discussion).
 <ol>
  <li><p>If <var>event</var>'s <a>stop propagation flag</a> is set, then return.
 
- <li><p>Let <var>listeners</var> be a new <a>list</a>.
+ <li><p>Let <var>listeners</var> be a new <a for=/>list</a>.
```

Comment:
Explicit for, which requires for="" always if the source has it.

I think link defaults is good and honestly we could enable link defaults to infra for all of WHATWG on things like list/string.

Anchors is the bad one.